src/Uno.UI/Microsoft/UI/Xaml/Controls/SplitButton/SplitButton.cs

```diff
@@ -0,0 +1,405 @@
+					}
+					else if (secondaryButton.IsPressed)
+					{
+						VisualStateManager.GoToState(this, "CheckedSecondaryPressed", useTransitions);
+					}
+					else if (secondaryButton.IsPointerOver)
+					{
+						VisualStateManager.GoToState(this, "CheckedSecondaryPointerOver", useTransitions);
+					}
+					else
+					{
+						VisualStateManager.GoToState(this, "Checked", useTransitions);
+					}
+				}
+				else
+				{
+					if (m_lastPointerDeviceType == PointerDeviceType.Touch || m_isKeyDown)
+					{
+						if (primaryButton.IsPressed || secondaryButton.IsPressed || m_isKeyDown)
+						{
+							VisualStateManager.GoToState(this, "TouchPressed", useTransitions);
+						}
+						else
+						{
+							VisualStateManager.GoToState(this, "Normal", useTransitions);
+						}
+					}
+					else if (primaryButton.IsPressed)
+					{
+						VisualStateManager.GoToState(this, "PrimaryPressed", useTransitions);
+					}
+					else if (primaryButton.IsPointerOver)
+					{
+						VisualStateManager.GoToState(this, "PrimaryPointerOver", useTransitions);
+					}
+					else if (secondaryButton.IsPressed)
+					{
+						VisualStateManager.GoToState(this, "SecondaryPressed", useTransitions);
+					}
+					else if (secondaryButton.IsPointerOver)
+					{
+						VisualStateManager.GoToState(this, "SecondaryPointerOver", useTransitions);
+					}
+					else
+					{
+						VisualStateManager.GoToState(this, "Normal", useTransitions);
+					}
+				}
+			}
+		}
+
+		internal void OpenFlyout()
+		{
+			var flyout = Flyout;
+			if (flyout != null)
+			{
+				if (SharedHelpers.IsFlyoutShowOptionsAvailable())
+				{
+					FlyoutShowOptions options = new FlyoutShowOptions();
+					options.Placement = FlyoutPlacementMode.BottomEdgeAlignedLeft;
+					flyout.ShowAt(this, options);
+				}
+				else
+				{
+					flyout.ShowAt(this);
+				}
+			}
+		}
+
+		internal void CloseFlyout()
+		{
+			var flyout = Flyout;
+			if (flyout != null)
+			{
+				flyout.Hide();
+			}
+		}
+
+		private void OnFlyoutOpened(object sender, object args)
+		{
+			m_isFlyoutOpen = true;
+			UpdateVisualStates();
+			SharedHelpers.RaiseAutomationPropertyChangedEvent(this, ExpandCollapseState.Collapsed, ExpandCollapseState.Expanded);
+		}
+
+		private void OnFlyoutClosed(object sender, object args)
+		{
+			m_isFlyoutOpen = false;
+			UpdateVisualStates();
+			SharedHelpers.RaiseAutomationPropertyChangedEvent(this, ExpandCollapseState.Expanded, ExpandCollapseState.Collapsed);
+		}
+
+		private void OnFlyoutPlacementChanged(DependencyObject sender, DependencyProperty dp)
+		{
+			UpdateVisualStates();
+		}
+
+		internal virtual void OnClickPrimary(object sender, RoutedEventArgs args)
+		{
+			var eventArgs = new SplitButtonClickEventArgs();
+
+			Click?.Invoke(this, eventArgs);
+
+			AutomationPeer peer = FrameworkElementAutomationPeer.FromElement(this);
+			if (peer != null)
+			{
+				//peer.RaiseAutomationEvent(AutomationEvents.InvokePatternOnInvoked);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5246693)